poedit.appdata.xml

```diff
@@ -1,31 +1,76 @@
 <?xml version="1.0" encoding="utf-8"?>
-<components version="0.11">
-  <component type="desktop-application">
+<component type="desktop">
     <id>poedit.desktop</id>
```

Comment:
Since we're updating this for conformance with AppStream spec, it would be nice to update the ID as well and use `net.poedit.Poedit`.

Reply:
Yeah that would be perfect! Will take care of that :+1: